client/my-sites/upgrades/navigation.jsx

```diff
@@ -123,14 +118,12 @@ var UpgradesNavigation = React.createClass( {
 		var items;
 
 		if ( this.props.selectedSite.jetpack ) {
-			items = [ 'Plans', 'My Upgrades' ];
-		} else if ( config.isEnabled( 'upgrades/domain-management/list' ) ) {
-			items = [ 'Plans', 'Domains', 'Email', 'My Upgrades' ];
+			items = [ 'Plans', 'My Purchases' ];
+		} else {
+			items = [ 'Plans', 'Domains', 'Email', 'My Purchases' ];
 			if ( config.isEnabled( 'upgrades/purchases/list' ) ) {
```

Comment:
@gziolo Good point, I hadn't considered that. None of the routes in `/my-sites/upgrades` are accessible in the Mac App environment, so that shouldn't be an issue here. There are only two cases we need to handle in navigation now: Jetpack and .com sites, the former of which shoudln't see links to domains or email.